applications/conceptual-model-editor/src/action/actions-react-binding.tsx

```diff
@@ -422,14 +484,102 @@ function createActionsContext(
+      alert("Anchoring node");
+    },
+    onShowSelectionActions: (source, flowPosition) => {
+      console.log("Application.onShowSelectionActions", { source, flowPosition });
+      diagram.actions().openCanvasToolbar(source, flowPosition, "NODE-SELECTION-ACTIONS-SECONDARY-TOOLBAR");
+    },
+    onLayoutSelection: () => {
+      alert("TODO: currently does nothing (In future - Opens layouting menu - 3 buttons - alignments + layouting)");
+    },
+    onCreateGroup: () => {
+      alert("TODO: currently does nothing (In future - Creating group)");
+    },
+    onShowExpandSelection: () => {
+      alert("TODO: currently does nothing (In future - Showing expansion dialog)");
+    },
+    onShowFilterSelection: () => {
+      alert("TODO: currently does nothing (In future - Showing filter dialog)");
+    },
+    onCanvasOpenCreateClassDialog: (sourceClassNode, positionToPlaceClassOn) => {
+      diagram.actions().closeCanvasToolbar();
+      // TODO: Maybe all of this should be in action - openCreateClassDialogWithModelDerivedFromClassAction
+
+      let model = findSourceModelOfEntity(sourceClassNode.externalIdentifier, graph.models);
+      if (model === null || !(model instanceof InMemorySemanticModel)) {
+        // Take the first model in memory model ... It is done the same way in create connection dialog
+        // TODO: Note that this causes crash, if there is no InMemorySemanticModel at all - I will make issue
+        filterInMemoryModels([...graph.models.values()])[0];
+        model = filterInMemoryModels([...graph.models.values()])?.[0] ?? null;
+      }
+
+      if (model === null) {
+        notifications.error("Can't find InMemorySemanticModel to put the association in");
+        return;
+      }
 
+      placePositionOnGrid(positionToPlaceClassOn, configuration().xSnapGrid, configuration().ySnapGrid);
+
+      // TODO: The way of creating dialog changed
+
+      // const onConfirm = (state: CreateClassDialogState) => {
+      //   createClassAction(notifications, graph, model as InMemorySemanticModel, positionToPlaceClassOn, state);
+      //   // If (Only If!) we want to add in the option to open CreateConnection dialog right after then we need 2 things:
+      //   //            1) Get the result of createClassAction. There are 2 ways to do this.
+      //   //                    a) Wait for dialog to finish here, but we also need to get the result and the "classes" variable is not updated, so we would have to get result somehow
+      //   //                    b) Pass in callback
+      //   //            2) Currently when we open new dialog here it is not opened - The issue is because when the current dialog (so CreateEditClass) is closed it also closes
+      //   //               all the following which are to be put in the stack - the problem is that we are not using the old state but just setting the new one
+      // };
+
+      // dialogs?.openDialog(createEditClassDialog(model, options.language, onConfirm));
+
```

Comment:
Basically the idea here is to open the dialog to create association between the node from which we dragged to canvas and the newly created one.

I describe the issues with this feature in the comment.

About the last part mentioning not using the old state I am not so sure about, but it seemed to me as possibility, since when we open multiple dialogs outside the onConfirm, they are opened correctly (at least if my memory serves me well)